packages/core/stories/button.doc.stories.mdx

```diff
@@ -57,6 +57,12 @@ Button contains three variants `cta`, `primary` and `secondary`. If not specifie
 
 Icons can be put before or after text, or used alone.
 
+#### Design considerations
+
+- An icon should be put before the text if it is further describing the button's action, such as upload, submit, play, done. This can be useful in busy environments where an additional description can improve speed to locate actions.
```

Comment:
```suggestion
- An icon should be put before the text if it is further describing the button's action, such as upload, submit, play, or done. This can be useful in busy environments where an additional description can improve speed in locating actions.
```

Reply:
I think 'done' is an example of a button label. I will update these to title case, with quote marks, to reflect that.